src/test/scala/edu/ie3/simona/agent/grid/ThermalGridIT.scala

```diff
@@ -67,12 +67,12 @@ class ThermalGridIT
     with ThermalHouseTestData
     with AnyWordSpecLike
     with should.Matchers
+    with QuantityMatchers
     with EmInputTestData
     with MockitoSugar
     with DefaultTestData
     with TestSpawnerTyped {
-  private implicit val quantityTolerance: Double = 1e-10d
-  protected val temperatureTolerance: Double = 0.01
+  protected given temperatureTolerance: Double = 0.01
```

Comment:
This is quite dangerous, I've already fallen into that trap myself. Now this tolerance applies to all doubles, e.g. also the power, qDot, etc. This is not what we want.

Reply:
But if you have an approach to get rid of the explicit tempTolerance, I would be very grateful.

Reply:
If we want remove the tolerance, we have to change the default tolerance value in the `equalWithTolerance` method. I found a way, where the temperature tolerance will only be used for checking temperature values.